### PR TITLE
Refactor preparacion cuentas comafi

### PR DESCRIPTION
**Pull request by guillermoojeda**
This PR refactors the function preparacion_cuentas_comafi. The refactored function is still in a separate file and does not intervene in the original process, but will do in further PRs.

Tests went OK.

### CODE REVIEW

**Comment by L-Giordano** [comment]
I think that maybe the three modules: [fill_data.py](https://github.com/alquimistas-org/subida_cartera/blob/refactor_preparacion_cuentas_comafi/src/functions/fill_data.py), [preparacion_cuentas_comafi.py](https://github.com/alquimistas-org/subida_cartera/blob/refactor_preparacion_cuentas_comafi/src/functions/preparacion_cuentas_comafi.py), [replace_invalid_chars.py](https://github.com/alquimistas-org/subida_cartera/blob/refactor_preparacion_cuentas_comafi/src/functions/replace_invalid_chars.py) could be grouped in a single module, because each of these three modules contains only one function, and all that functions are helpers to the [subida.py](https://github.com/alquimistas-org/subida_cartera/blob/refactor_preparacion_cuentas_comafi/src/subida.py) main function.

**Review by matiasbertani — CHANGES_REQUESTED** [review]
Nice function wrapping! `preparacion_cuentas_comafi` looks shorter and is more understandable what is doing. 

I would like all the funciton be in only one module for now, exactly as @natilou did it in this PR https://github.com/alquimistas-org/subida_cartera/pull/8. In this way we are being more consistency each other. I think we had to follow tha path Natri choose, it seem to be very clean.

Also I noticed that in the new module you created you're adding imports that are not use it in the module. Rigth now we don't have a CI checking flake8, but it is planned in this week, and flake is not going to like that kind of imports jajaja

To summarize, the code look good to mee, i only ask you to change the estructure to follow Nati's one

**Comment by guillermoojeda** [comment]
Thanks for the suggestions. The changes suggested have been implemented.

File: src/preparacion_cuentas_comafi.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    # iterear un loop por los subcliente a traves de un grouby
+    for name, df_sub in df_os.groupby('subcliente'):
+        print(f'Ecribiendo: {name}.csv')
+        df_sub = df_sub.drop('subcliente', inplace=False, axis=1)
+        df_sub.to_csv(f'{name_folder}/{name}.csv', sep=';', encoding='ANSI', index=False)
+    return ' '
+
+
+def replace_invalid_chars(dataframe):
+    characters = ['#', 'Ð', 'ð', '&']
```
Comment (matiasbertani) [inline src/preparacion_cuentas_comafi.py:52]: We could this in the future migrate it to `constants.py` module

File: src/preparacion_cuentas_comafi.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    df = df.rename(columns=UTIL_COLS_COMAFI)
+
+    # reemplazo de valores nulos
+    df.loc[df['provincia'].isna(), 'provincia'] = '0'
+    # reemplazo de 'ñ' en nombres
+    replace_invalid_chars(df)
+    print('\n')
+    print('\nComenzando escritura de archivos..\n\n')
+
+    fill_data(df, df_os)
```
Comment (matiasbertani) [inline src/preparacion_cuentas_comafi.py:37]: I don't want to to change this, but., I only leave for document and remember in the future. we could return `df_os` riectly from here, and also create it inside this method. We could name the method `build_df_os( )`the only change needed is move the line [24](https://github.com/alquimistas-org/subida_cartera/pull/9/files#diff-e190420606fa8cc07fa4a12db3e90babfb8bd77c818cb057320e694de41761f1R24) inside `fille_data`

**Review by matiasbertani — APPROVED** [review]
lgtm! great wrapping

File: src/preparacion_cuentas_comafi.py (hunk 2 above)
Comment (guillermoojeda) [reply src/preparacion_cuentas_comafi.py:37]: OK, on it.

**Review by matiasbertani — APPROVED** [review]
lgtm! Thanks guille
